src/form.ts

```diff
@@ -124,6 +125,21 @@ export class Form<T> {
     return changes;
   }
 
+  // Return an object with the fields that have been touched
+  get touched(): Partial<T> {
```

Comment:
I have a bit of an issue with the name. So far `touched` was a boolean across the code base and at `form.touched` I would expect a boolean indicating if any of the fields have been touched instead of an object (which would evaluate to truthy when empty because it's an object). How about `touchedValues`?